src/components/network/line-layer.js

```diff
@@ -418,9 +498,19 @@ class LineLayer extends CompositeLayer {
                             this.props.lineFullPath
                         ),
                     getColor: (arrow) =>
-                        getLineColor(arrow.line, color, this.props),
+                        getLineColor(
+                            arrow.line,
+                            nominalVoltageColor,
+                            this.props
+                        ),
                     getSize: 700,
-                    getSpeedFactor: 3,
+                    getSpeedFactor: (arrow) =>
+                        getArrowSpeedFactor(
+                            getArrowSpeed(
+                                arrow.line,
+                                this.props.lineFlowAlertThreshold
```

Comment:
parameter 'this.props.lineFlowAlertThreshold' not used

Reply:
Fixed